sphinx_gherkindoc/cli.py

```diff
@@ -185,6 +190,11 @@ def main() -> None:
             "is also included."
         ),
     )
+    parser.add_argument(
+        "--pytest-parser",
+        action="store_true",
+        help="Use the pytest-bdd parser rather than the behave parser.",
+    )
```

Comment:
I think it'd make more sense to have this be like `--parser pytest-bdd` and have `parser` just default to `behave`.